common/user-clients.ts

```diff
@@ -16,4 +16,5 @@ export type UserClient = {
   votes_count?: number;
   awards?: any[];
   custom_goals?: CustomGoal[];
+  challenge_team?: string;
```

Comment:
This will probably look different after the schema changes, eg:

```
challenge_enrollments?: {
  ...
  team: string,
}[];
```

But this looks good for now!